instrumentation/grpctrace/interceptor.go

```diff
@@ -450,10 +466,10 @@ func peerInfoFromContext(ctx context.Context) []kv.KeyValue {
 	return peerInfoFromTarget(p.Addr.String())
 }
 
-var fullMethodRegexp = regexp.MustCompile(`^\/?(?:\S+\.)?(\S+)\/\S+$`)
+var methodRegexp = regexp.MustCompile(`^\/?(?:\S+\.)?(?:\S+)\/(\S+)$`)
 
-func serviceFromFullMethod(method string) string {
-	match := fullMethodRegexp.FindStringSubmatch(method)
+func methodFromFullName(method string) string {
+	match := methodRegexp.FindStringSubmatch(method)
 	if len(match) == 0 {
 		return ""
 	}
```

Comment:
Rather than a regex, something like

```go
	lastSlash := strings.LastIndexByte(method, '/')
	if lastSlash == -1 {
		return ""
	}
        return method[lastSlash+1:]
```

may be faster.